.github/codeql/java-custom-queries.qls

```diff
@@ -0,0 +1,10 @@
+- import: codeql-suites/java-code-scanning.qls
+  from: codeql-java
+- exclude:
+    id:
+    - java/tainted-numeric-cast
+    - java/polynomial-redos
+    - java/path-injection
+    - java/ssrf
```

Comment:
Is there some documentation for these? What do they do? What else is enabled?

Reply:
I'll add a readme file with links to the docs.
* https://codeql.github.com/codeql-query-help/java/
* https://codeql.github.com/codeql-query-help/java-cwe/